logging/bin/secrets-include.sh

```diff
@@ -14,7 +14,7 @@ function create_secret_from_file {
 
   if [ -z "$(kubectl -n $LOG_NS get secret $secret_name -o name 2>/dev/null)" ]; then
 
-    log_debug "Will attempt to create secret [$secret_name]"
+    # log_debug "Will attempt to create secret [$secret_name]"
 
```

Comment:
I vote for keeping this debug comment in.

Reply:
I'll go ahead and keep them in and change the verbiage a bit from "Will attempt to create" to "Creating."